packages/hub/Dockerfile

```
@@ -8,6 +8,13 @@ COPY dist/sql node_modules/graphile-worker/sql
 # This includes the discord bot image assets that are hosted by the hub
 COPY dist/services packages/hub/services
 
+# Copy Prisma necessities
+COPY dist/@prisma/client node_modules/.prisma/client
```

Is this a typo? If so we might not need the subsequent COPY instructions for `dist/.prisma/<somefile>`?

```suggestion
COPY dist/.prisma/client node_modules/.prisma/client
```